.ci/setup_ci.sh

```diff
@@ -2,7 +2,7 @@
 
 set -x
 
-if [ "$RESTY_IMAGE_TAG" != "xenial" ] && [ "$RESTY_IMAGE_BASE" != "alpine" ]; then
+if [ "$RESTY_IMAGE_TAG" != "xenial" ] && [ "$RESTY_IMAGE_TAG" != "16.04" ] && [ "$RESTY_IMAGE_BASE" != "alpine" ]; then
```

Comment:
I'm having a hard time interpreting the intent here.
This seems to be meant to be a no-op on xenial or alpine, but it's unclear why.

A comment here would be appreciated.

Reply:
The setup_ci.sh file isn't a great name (anymore).

The file sets up ci systems such that they're capable of building / releasing arm assets which we only do for alpine and xenial